src/main/java/com/powsybl/openloadflow/sensi/AcSensitivityAnalysis.java

```diff
@@ -210,6 +216,17 @@ public void analyse(Network network, List<PropagatedContingency> contingencies,
                 slackParticipationByBus = Collections.singletonMap(lfNetwork.getSlackBus(), -1d);
             }
 
+            // if we have at least one bus target voltage linked to a ratio tap changer, we have to rebuild the AC equation
+            // system obtained just before the transformer steps rounding.
+            if (hasTransformerBusTargetVoltage) {
+                for (LfBus bus : lfNetwork.getBuses()) {
```

Comment:
I am not sure how it will behave on contingencies : 
I guess the mode will be turned `OFF` again ? so maybe it does not work for the post contingency sensitivities ?

Reply:
I will add a test but it will be similar to the generator behaviour.